sideci_demo/rubocop_demo.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# GitHub Repository
+class Repository
+  attr_reader :client
+
+  def initialize(owner, name)
+    @owner = owner
+    @name = name
+    @client = Github.new
+  end
+
+  def releases
+    repository&.releases.map(&:name)
```

Comment:
__[staging][RuboCop]__

Lint/SafeNavigationChain: Do not chain ordinary method call after safe navigation operator.


<!-- Comment by [SideCI](https://sideci.com) -->